src/NuGet.Core/NuGet.Packaging/PackageCreation/Resources/NuGetResources.resx

```diff
@@ -212,4 +212,12 @@
     <value>The element 'icon' cannot be empty.</value>
     <comment>Please, don't localize icon</comment>
   </data>
+  <data name="IconInvalidExtension" xml:space="preserve">
+    <value>The icon element '{0}' has an invalid file extension. Valid options are .png, .jpg or .jpeg.</value>
+    <comment>0 - Icon entry in the nuspec</comment>
+  </data>
+  <data name="IconEmptyExtension" xml:space="preserve">
+    <value>The icon element '{0}' file extension is empty. Valid options are .png, .jpg or .jpeg.</value>
```

Comment:
This reads a little strangely for me. Maybe `... is missing a file extension`?

Reply:
I will remove that string resource and use only one for invalid icon file extension.